governance.md

```diff
@@ -0,0 +1,258 @@
+
+The Lead Maintainer may nominate a successor, subject to acceptance by a majority of Core Developers.
+If the Lead Maintainer is unable to appoint a successor, the nominated successor is not approved, or the Lead Maintainer is recalled, the Core Developers will choose a successor by two-thirds vote.
+If no Lead Maintainer candidate receives two-thirds of the votes of all active Core Developers after three rounds of voting, the Core Developers shall propose the Lead Maintainer candidates to the NumFOCUS board, who will then make the final decision.
+
+In extraordinary circumstances, if the Lead Maintainer has lost the confidence of the Core Developers, a supermajority of two-thirds of all active Core Developers (not including the Lead Maintainer) may vote to recall the Lead Maintainer.
+This is intended as a last resort, after exhausting all other reasonable efforts to resolve the situation.
+If recalled, the Lead Maintainer would remain a Core Developer, with any further status changes happening as with any other member of that Team.
+
+
+### Core Developers
+
+The Project's Core Developers will consist of Project Contributors who have produced contributions that are substantial in quality and quantity, and sustained over at least one year.
+The overall role of the Core Developers is to ensure, through working with the Lead Maintainer and taking input from the Community, the long-term well-being of the project, both technically and otherwise.
+Further, it is expected that because of the quality and quantity of their contributions and their expert knowledge of the Project Software, the Core Developers will provide guidance, both technical and in terms of project direction, to less experienced contributors.
+
+The Core Developers will be members of the "Core Developers" Team in the Spyder-IDE Organization, which is granted the Write role on the primary [`spyder`](https://github.com/spyder-ide/spyder) and [`spyder-kernels`](https://github.com/spyder-ide/spyder-kernels) repositories.
+Users must be Core Developers to be granted Write-level access and above on these repositories, as well as Admin-level access on other repositories in the Spyder-IDE organization.
+It is the Core Developers that vote on specific, major, high-level project governance issues as specified in this document.
+Additionally, the Core Developers have primary responsibility for guiding the code review process and merging pull requests, and for administering and maintaining any key services and infrastructure on which the project relies.
+
+To become eligible for becoming a Core Developer, an individual must be a Project Contributor who has produced contributions that are substantial in quality and quantity, and sustained over at least one year.
+Potential Core Developers are nominated by an Core Developers and voted upon by the Core Developers after asking if the potential Core Developer is interested and willing to serve in that capacity.
```

Comment:
```suggestion
Potential Core Developers are nominated by a Core Developer and voted upon by the Core Developers after asking if the potential Core Developer is interested and willing to serve in that capacity.
```

Reply:
Thanks, I fixed that and revised the sentence a bit to make it shorter, add a missing comma and reduce repetition.